### DISCUSSION TITLE
Projeto blazor .NET 8

### DISCUSSION THREAD

**Discussion by IgorDias1998** [balta-io/forum#448] [Tira Dúvidas] [answered]
Boa noite pessoal, uma dúvida que me surgiu durante o projeto do curso blazor, seria como exibir as informações criadas apenas por determinada pessoa autenticada. Por exemplo, quando criar um novo produto, outro login consegue ver e manipular as mesmas informações.
Então o correto seria cada usuário autenticado ter acesso apenas aos seus próprios dados criados. Alguém consegue ajudar com isso?

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @IgorDias1998, como vai?

Dá uma conferida neste post aqui e veja se te ajuda:
https://github.com/balta-io/forum/discussions/428
